### DISCUSSION TITLE
익스텐션에서 태그 목록 조회 요청 대량 발생 예상

### DISCUSSION THREAD

**Discussion by dmdgpdi** [Kernel360/F2-BAGUNI#65] [이슈 / 버그]
@enigsuss  @Gyaak @kimminkyeu @sangwonsheep @dmdgpdi 

현재 탭에서 확장 프로그램을 닫았다 열었을 때, 새롭게 요청이 보내집니다. 
(사용자가 10개 탭에서 확장 프로그램을 열었을 때, 10개의 태그 목록 조회 요청이 발생합니다.)
이걸 어떻게 하면 좋을지 의견을 나눠봐야 할 것 같습니다.

**Comment by sangwonsheep** [comment]
서면으로 해당 문제에 대해 들었습니다. 정상적인 요청이라고 생각합니다.
추후 봇이 1초에 엄청 많은 요청을 보내게 되는 경우를 고려해야 합니다.
